theme/decouplededitor.css

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
```

Comment:
Why does it exist? Maybe a comment here? Or can we remove it?

Reply:
Especially that I can see something like this:

```
 [ThemeImporter] Failed to find "/workspace/ckeditor5/packages/ckeditor5-theme-lark/theme/ckeditor5-editor-decoupled/decouplededitor.c
```

Reply:
Looks like a post–copy&paste orphan.

Reply:
I removed it and all seems to work fine.